templates/java/org/yarp/Loader.java.erb

```diff
@@ -257,19 +257,19 @@ public class Loader {
             case <%= index + 1 %>:
             <%-
             params = node.needs_serialized_length? ? ["buffer.getInt()"] : []
```

Comment:
`params` --> `fields`

Reply:
For this one I left it in because it's being used as the params to the function being described.